### PR TITLE
 Fix alpha channel handling in xBRZ texture filter shader

### PR DESCRIPTION
**Pull request by BreadFish64**
I replaced `DistYCbCr` with the algorithm used in Citra which has proper alpha weighting.

### CODE REVIEW

File: GPU/Vulkan/TextureCacheVulkan.cpp [hunk 1]
```diff
@@ -84,15 +80,16 @@ float reduce(vec4 color) {
 }
 
 float DistYCbCr(vec4 pixA, vec4 pixB) {
-	const vec3 w = vec3(0.2627, 0.6780, 0.0593);
-	const float scaleB = 0.5 / (1.0 - w.b);
-	const float scaleR = 0.5 / (1.0 - w.r);
+	// https://en.wikipedia.org/wiki/YCbCr#ITU-R_BT.2020_conversion
+	const vec3 K = vec3(0.2627, 0.6780, 0.0593);
+	const mat3 MATRIX = mat3(K, 
+							-.5 * K.r / (1.0 - K.b),	-.5 * K.g / (1.0 - K.b),	.5,
```
Comment (unknownbrackets) [inline GPU/Vulkan/TextureCacheVulkan.cpp:86]: nit: I always recommend spaces for alignment, tabs for indent.  Tabs for alignment is where things get messy.

-[Unknown]

**Review by unknownbrackets — APPROVED** [review]

File: GPU/Vulkan/TextureCacheVulkan.cpp (hunk 1 above)
Comment (BreadFish64) [reply GPU/Vulkan/TextureCacheVulkan.cpp:86]: I do too, but the rest of this raw string of formatted with tabs. Do you want me to replace all of them with spaces?

Comment (unknownbrackets) [reply GPU/Vulkan/TextureCacheVulkan.cpp:86]: Ah, I guess because it came from DeSmuME then.  Okay, then.

-[Unknown]

**Comment by hrydgard** [comment]
Cool!

**Pull request merged**